tests/tests_aqua.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import requirements_parser
+import os
+
+requirements_parser = requirements_parser.RequirementsParser()
+
+
+class MyTestCase(unittest.TestCase):
+
+    def test_requirements_num(self):
```

Comment:
where is the use case without txt, with " ", with /n, without requirment.txt, without -r...